authentication/rosa-hcp-managing-breakglass-cred.adoc

```diff
@@ -0,0 +1,22 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="rosa-hcp-managing-breakglass-cred"]
+= Managing a break glass credential for a {hcp-title} cluster
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-hcp-managing-breakglass-cred
+
+toc::[]
+
+As a {hcp-title} cluster owner, you can use the break glass credential to gain temporary administrative client credentials to your clusters configured with custom OpenID Connect (OIDC) token issuers. The temporary credentials are created in a cluster-admin `kubeconfig` file. The `kubeconfig` file contains information about the cluster that the CLI uses to connect a client to the correct cluster and API server. You can use the newly generated `kubeconfig` file to allow access to the cluster in the following situations:
+
+* Between the point at which the cluster is created and when the OIDC configuration is complete and authentication can occur through normal flows.
+* If the OIDC compliant identity provider becomes unavailable or misconfigured.
+
+
+//To use this break glass credential, you must create a {hcp-title} cluster with an external authentication provider enabled. For more information, see xref:../rosa_hcp/rosa-hcp-sts-creating-a-cluster-ext-auth.adoc#rosa-hcp-sts-creating-a-cluster-ext-auth[Creating ROSA with HCP clusters with external authentication].
```

Comment:
Is this a comment?

Reply:
That's a placeholder for a comment that I was going to add but have decided not to. I'll remove it now-thanks!